customfieldnames.civix.php

```diff
@@ -346,3 +346,166 @@ function _customfieldnames_civix_civicrm_alterSettingsFolders(&$metaDataFolders
     $metaDataFolders[] = $settingsDir;
   }
 }
+
+/**
+ * Implements hook_civicrm_buildForm().
+ *
+ * @param string $formName
+ * @param CRM_Core_Form $form
+ */
+function _customfieldnames_civix_civicrm_buildForm($formName, &$form) {
+  $templatePath = realpath(dirname(__FILE__)."/templates");
+  if( ($formName == 'CRM_Custom_Form_Group') && ($form->getAction() == CRM_Core_Action::ADD) ) {
+    $templatePath = realpath(dirname(__FILE__)."/templates");
+    $form->add('text', 'table_name', ts('Set Table Name'), '', FALSE);
+    $form->assign('table_name', TRUE);
+    $form->add('text', 'name', ts('Set Group Name'), '', FALSE);
+    $form->assign('name', TRUE);
+    
+    CRM_Core_Region::instance('page-body')->add(array(
+      'template' => "{$templatePath}/customgroup.tpl"
+    ));
+  }
+  if ( ($formName == 'CRM_Custom_Form_Field') && ($form->getAction() == CRM_Core_Action::ADD) ) {
+    $form->add('text', 'column_name', ts('Set Column Name'), '', FALSE);
```

Comment:
label "Column Name"

Reply:
change last param to "TRUE" to make this required. do for all that follow.

Reply:
Changed the label and set the reuired property to "TRUE" for all the custom fields.